integration-tests/deployment/clo/testdata/chain_writer_nodes.json

```diff
@@ -0,0 +1,1446 @@
+[
```

Comment:
testdata is just the output of `fmscli get nodeOperators`.  https://github.com/smartcontractkit/feeds-manager/tree/develop?tab=readme-ov-file#chainlink-orchestrator-api-client

this kind of data will be input to the testnet deployment